policy_set.go

```diff
@@ -524,6 +582,16 @@ func (o PolicySetRemoveWorkspacesOptions) valid() error {
 	return nil
 }
 
+func (o PolicySetRemoveWorkspaceExclusionsOptions) valid() error {
+	if o.WorkspaceExclusions == nil {
+		return ErrWorkspacesRequired
+	}
+	if len(o.WorkspaceExclusions) == 0 {
+		return ErrWorkspaceMinLimit
+	}
```

Comment:
💅 Just pointing out this is slightly redundant since len(nil) == 0 but I know the pattern is probably elsewhere.

```suggestion
	if len(o.WorkspaceExclusions) == 0 {
		return ErrWorkspacesRequired
	}
```

Reply:
I saw this redundancy but only kept it due to repeated pattern. I can clean this up.

Reply:
@brandonc We do not need the test for https://github.com/hashicorp/go-tfe/blob/hs26gill/TF-7895-Add-delete-policy-sets-relationships-exclusions/policy_set_integration_test.go#L870 as well then right?

Reply:
I think in the future the "required" error is sufficient when you provide either 0 items or nil, but I don't mind keeping it consistent!

Reply:
Sounds good, thanks!